solidity/contracts/stubs/RewardsStub.sol

```diff
@@ -17,14 +16,15 @@ contract RewardsStub is Rewards {
     ) public Rewards(
         _token,
         _firstIntervalStart,
-        _intervalWeights
+        _intervalWeights,
+        1 days, // term length
```

Comment:
Good point. `Rewards` contract was hardcoding the `_termLength` but unit tests want to use a different value. Instead of having `setTermLength`, I'll just pass this value to `RewardsStub` constructor since it is required by `Rewards` contract constructor anyway.